src/plugins/saved_objects_management/server/routes/scroll_count.ts

```diff
@@ -39,34 +39,65 @@ export const registerScrollForCountRoute = (router: IRouter) => {
       validate: {
         body: schema.object({
           typesToInclude: schema.arrayOf(schema.string()),
+          namespacesToInclude: schema.maybe(schema.arrayOf(schema.string())),
           searchString: schema.maybe(schema.string()),
         }),
       },
     },
     router.handleLegacyErrors(async (context, req, res) => {
       const { client } = context.core.savedObjects;
+      const namespaces = [];
+      if (req.body.namespacesToInclude && req.body.namespacesToInclude.length > 0) {
+        req.body.namespacesToInclude.forEach((ns) => {
+          if (ns === null) {
+            namespaces.push('default');
+          } else {
+            namespaces.push(ns);
+          }
+        });
+      }
 
       const findOptions: SavedObjectsFindOptions = {
+        ...req.body.params,
```

Comment:
Why do we need this?

Reply:
Legacy of the first `filterRegistry` implementation that I didn't remove. Thanks for catching!